cmd/client.go

```diff
@@ -92,14 +92,24 @@ func (cli *Client) GetJobSpecs(c *clipkg.Context) error {
 func (cli *Client) CreateJobSpec(c *clipkg.Context) error {
 	cfg := cli.Config
 	if !c.Args().Present() {
-		return cli.errorOut(errors.New("Must pass in job JSON"))
+		return cli.errorOut(errors.New("Must pass in JSON/filepath"))
+	}
+	var buf *bytes.Buffer
+	if c.Args().First()[0] != '{' {
```

Comment:
Let's have a more robust json check. We already have `gjson` so you can do the following:
https://github.com/tidwall/gjson#validate-json

```
if !gjson.Valid(json) {
	return errors.New("invalid json")
}
```